Auto update failing for exfatprogs

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [exfatprogs](https://github.com/termux/termux-packages/blob/8705fc7ea560fe222019766bf9856adcff03b876/packages/exfatprogs) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating exfatprogs [Current version: 1.3.2, github/latest-release-tag]
INFO: package being updated to 1.4.0.
Downloading https://github.com/exfatprogs/exfatprogs/releases/download/1.4.0/exfatprogs-1.4.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  596k  100  596k    0     0  2799k      0 --:--:-- --:--:-- --:--:-- 2799k

You are about to commit these changes:

--------------------
diff --git a/packages/exfatprogs/build.sh b/packages/exfatprogs/build.sh
index 6688f1da59..ed3cf4364f 100644
--- a/packages/exfatprogs/build.sh
+++ b/packages/exfatprogs/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE="https://github.com/exfatprogs/exfatprogs"
 TERMUX_PKG_DESCRIPTION="exFAT filesystem userspace utilities"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.3.2"
+TERMUX_PKG_VERSION="1.4.0"
 TERMUX_PKG_SRCURL="https://github.com/exfatprogs/exfatprogs/releases/download/${TERMUX_PKG_VERSION}/exfatprogs-${TERMUX_PKG_VERSION}.tar.gz"
-TERMUX_PKG_SHA256=0c5d445947df781f90ba6bfddbd323bd6324c78a51fe75380a6ce2238c3cbcce
+TERMUX_PKG_SHA256=3bc1091b2b43be7ca49517d21794a6b7fd1423abc6413398bb9475ce7eb6469f
 TERMUX_PKG_AUTO_UPDATE=true
 
 termux_step_pre_configure() {
--------------------

bump(main/exfatprogs): 1.4.0


Not committing to Git!

INFO: Trying to build package.
INFO: Package exfatprogs exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
Loading restricted AppArmor profile...
termux - building exfatprogs for arch aarch64...
::group::INFO: Fetching repo metadata
[packages-cf.termux.dev-apt-termux-main]: Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
[packages-cf.termux.dev-apt-termux-main]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-main]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-main]: 
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]: 100 13134  100 13134    0     0  83246      0 --:--:-- --:--:-- --:--:-- 83656
[packages-cf.termux.dev-apt-termux-main]: Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
[packages-cf.termux.dev-apt-termux-main]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-main]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-main]: 
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]: 100   833  100   833    0     0   5362      0 --:--:-- --:--:-- --:--:--  5374
[packages-cf.termux.dev-apt-termux-main]: gpg: Signature made Wed 27 May 2026 07:07:52 AM UTC
[packages-cf.termux.dev-apt-termux-main]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-main]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-main]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
[packages-cf.termux.dev-apt-termux-root]: Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
[packages-cf.termux.dev-apt-termux-root]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-root]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-root]: 
[packages-cf.termux.dev-apt-termux-root]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]: 100 13320  100 13320    0     0  83695      0 --:--:-- --:--:-- --:--:-- 83773
[packages-cf.termux.dev-apt-termux-root]: Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
[packages-cf.termux.dev-apt-termux-root]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-root]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-root]: 
[packages-cf.termux.dev-apt-termux-root]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]: 100   833  100   833    0     0   4705      0 --:--:-- --:--:-- --:--:--  4732
[packages-cf.termux.dev-apt-termux-root]: gpg: Signature made Tue 26 May 2026 01:06:38 AM UTC
[packages-cf.termux.dev-apt-termux-root]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-root]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-root]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
[packages-cf.termux.dev-apt-termux-x11]: Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
[packages-cf.termux.dev-apt-termux-x11]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-x11]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-x11]: 
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]: 100 13120  100 13120    0     0  73727      0 --:--:-- --:--:-- --:--:-- 74124
[packages-cf.termux.dev-apt-termux-x11]: Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
[packages-cf.termux.dev-apt-termux-x11]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-x11]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-x11]: 
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]: 100   833  100   833    0     0   3089      0 --:--:-- --:--:-- --:--:--  3096
[packages-cf.termux.dev-apt-termux-x11]: gpg: Signature made Wed 27 May 2026 06:48:32 AM UTC
[packages-cf.termux.dev-apt-termux-x11]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-x11]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-x11]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
::endgroup::
Downloading exfatprogs@1.4.0 source from 'https://github.com/exfatprogs/exfatprogs/releases/download/1.4.0/exfatprogs-1.4.0.tar.gz' if necessary...
Downloading https://github.com/exfatprogs/exfatprogs/releases/download/1.4.0/exfatprogs-1.4.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  596k  100  596k    0     0  6700k      0 --:--:-- --:--:-- --:--:-- 6700k
configure: WARNING: unrecognized options: --disable-rpath, --disable-rpath-hack, --disable-nls
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for aarch64-linux-android-strip... llvm-strip
checking for a race-free mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking how to create a pax tar archive... gnutar
checking for aarch64-linux-android-gcc... aarch64-linux-android-clang
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... yes
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether aarch64-linux-android-clang accepts -g... yes
checking for aarch64-linux-android-clang option to enable C11 features... none needed
checking whether aarch64-linux-android-clang understands -c and -o together... yes
checking whether make supports the include directive... yes (GNU style)
checking dependency style of aarch64-linux-android-clang... none
checking whether make supports nested variables... (cached) yes
checking build system type... x86_64-pc-linux-gnu
checking host system type... aarch64-unknown-linux-android
checking how to print strings... printf
checking for a sed that does not truncate output... /usr/bin/sed
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for fgrep... /usr/bin/grep -F
checking for ld used by aarch64-linux-android-clang... ld.lld
checking if the linker (ld.lld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... llvm-nm
checking the name lister (llvm-nm) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 3145728
checking how to convert x86_64-pc-linux-gnu file names to aarch64-unknown-linux-android format... func_convert_file_noop
checking how to convert x86_64-pc-linux-gnu file names to toolchain format... func_convert_file_noop
checking for ld.lld option to reload object files... -r
checking for aarch64-linux-android-file... no
checking for file... file
configure: WARNING: using cross tools not prefixed with host triplet
checking for aarch64-linux-android-objdump... llvm-objdump
checking how to recognize dependent libraries... pass_all
checking for aarch64-linux-android-dlltool... no
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for aarch64-linux-android-ar... llvm-ar
checking for archiver @FILE support... @
checking for aarch64-linux-android-strip... (cached) llvm-strip
checking for aarch64-linux-android-ranlib... llvm-ranlib
checking command to parse llvm-nm output from aarch64-linux-android-clang object... ok
checking for sysroot... no
checking for a working dd... /usr/bin/dd
checking how to truncate binary pipes... /usr/bin/dd bs=4096 count=1
checking for aarch64-linux-android-mt... no
checking for mt... no
checking if : is a manifest tool... no
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if aarch64-linux-android-clang supports -fno-rtti -fno-exceptions... yes
checking for aarch64-linux-android-clang option to produce PIC... -fPIC -DPIC
checking if aarch64-linux-android-clang PIC flag -fPIC -DPIC works... yes
checking if aarch64-linux-android-clang static flag -static works... yes
checking if aarch64-linux-android-clang supports -c -o file.o... yes
checking if aarch64-linux-android-clang supports -c -o file.o... (cached) yes
checking whether the aarch64-linux-android-clang linker (ld.lld) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... Android linker
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... yes
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... no
checking whether byte ordering is bigendian... (cached) no
checking for fts.h... yes
checking for fts_open in -lfts... no
checking pkg-config is at least version 0.9.0... yes
checking for blkid >= 2.20... no
checking for blkid... no
configure: error: Package requirements (blkid) were not met:

Package 'blkid' not found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables EXT2_BLKID_CFLAGS
and EXT2_BLKID_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occurred when I last tried to update at 2026-05-27 08:08:41 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/26497312149">26497312149</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/exfatprogs/build.sh
+++ b/packages/exfatprogs/build.sh
@@ -2,12 +2,15 @@ TERMUX_PKG_HOMEPAGE="https://github.com/exfatprogs/exfatprogs"
 TERMUX_PKG_DESCRIPTION="exFAT filesystem userspace utilities"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.3.2"
-TERMUX_PKG_SRCURL="https://github.com/exfatprogs/exfatprogs/releases/download/${TERMUX_PKG_VERSION}/exfatprogs-${TERMUX_PKG_VERSION}.tar.gz"
-TERMUX_PKG_SHA256=0c5d445947df781f90ba6bfddbd323bd6324c78a51fe75380a6ce2238c3cbcce
+TERMUX_PKG_VERSION="1.4.0"
+TERMUX_PKG_SRCURL="https://github.com/exfatprogs/exfatprogs/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256=3340a80cede17a7f94497fdc54c69c0e6ea3625e314d960551967b046eed981c
 TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_DEPENDS="libblkid"
 
 termux_step_pre_configure() {
+	autoreconf -fiv
+
 	CFLAGS+=" -D_FILE_OFFSET_BITS=64"
 	CPPFLAGS+=" -D_FILE_OFFSET_BITS=64"
 }

